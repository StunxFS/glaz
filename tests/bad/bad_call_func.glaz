fn y(_a: i32) {}
fn z(_b: i32, _c: i32) {}

fn main() {
  y(10, 11);
  z(10);
}
