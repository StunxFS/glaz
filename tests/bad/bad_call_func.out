tests/bad/bad_call_func.glaz:5:4: error: too many arguments to function `y`
     | 
   5 |   y(10, 11);
     |   ^^^^^^^^^
     = note: expected 1 argument(s), not 2
tests/bad/bad_call_func.glaz:6:4: error: too few arguments to function `z`
     | 
   6 |   z(10);
     |   ^^^^^
     = note: expected 2 argument(s), not 1
glaz error: could not compile `bad_call_func`, aborting due to 2 previous errors.
